packages/web-components/src/components/va-process-list/va-process-list.scss

```diff
@@ -1,3 +1,38 @@
+:host([uswds]) ::slotted(li):before {
+  color: var(--v3-color-base-darkest);
+  border: 4px solid var(--v3-color-base-darkest);
+  background: var(--color-white);
+  box-shadow: 0 0 0 4px var(--color-white);
+  font-size: 22px;
+  font-weight: 700;
+}
+
+:host([uswds]) ::slotted(li.checkIcon):before {
+  color: var(--color-green);
+  border: 4px solid var(--color-green);
+  content: url(./green-check.svg);
+}
+
+:host([uswds]) ::slotted(li.activeIcon):before {
+  color: var(--color-primary);
+  border: 4px solid var(--color-primary);
+}
+
+:host([uswds]) ::slotted(li) {
+  border-left: 8px solid var(--color-blue-cool-lightest);
+}
+
+:host([uswds]) ::slotted(li:last-child) {
+  border-left: 0;
+  padding-left: calc(3.1rem + 8px);
+}
+
+:host(:not([uswds])){
```

Comment:
I noticed that the `:host(:not([uswds])){}` block in the CSS file doesn't have any styles associated with it. If it's not serving a specific purpose, we might consider removing it to keep our CSS lean and clear of unnecessary selectors.